third_party/rust-on-exercism/health-statistics.rs

```diff
@@ -2,6 +2,20 @@ pub struct User {
     name: String,
     age: u32,
     height: f32,
+    doctor_visits: usize,
```

Comment:
Is this intentionally different from `HealthReport::visit_count`?

Reply:
Yes--a patient has visited the doctor a given number of times, but a health report corresponds to a particular visit whose count it carries. Maybe `visit_index` could be used, but I thought that might suggest zero-indexing or that there's a vector of visits somewhere.

Reply:
The plural ("visits") suggests a count. If it's not zero-indexed, what is it -- a datestamp?

Reply:
It's probably least confusing if we just make these variable names match. I'll update them to both be `visit_count`.

Reply:
@domenukk, see above for your question about `usize` vs `u32`.